tests/test_multidict.py

```diff
@@ -37,212 +37,214 @@ def test_exposed_names(self):
+        assert sorted(d.values()) == [0, 1]
+        assert sorted(d.items()) == [('0', 0), ('1', 1)]
 
     def test_getone(self):
         d = self.make_dict([('key', 'value1')], key='value2')
-        self.assertEqual(d.getone('key'), 'value1')
-        self.assertEqual(d.get('key'), 'value1')
-        self.assertEqual(d['key'], 'value1')
+        assert d.getone('key') == 'value1'
+        assert d.get('key') == 'value1'
+        assert d['key'] == 'value1'
 
-        with self.assertRaisesRegex(KeyError, 'key2'):
+        with pytest.raises(KeyError) as excinfo:
             d['key2']
-        with self.assertRaisesRegex(KeyError, 'key2'):
+        assert 'key2' in str(excinfo.value)
+        with pytest.raises(KeyError) as exc2info:
             d.getone('key2')
+        assert 'key2' in str(exc2info.value)
 
-        self.assertEqual('default', d.getone('key2', 'default'))
+        assert 'default' == d.getone('key2', 'default')
 
     def test__iter__(self):
         d = self.make_dict([('key', 'one'), ('key2', 'two'), ('key', 3)])
-        self.assertEqual(['key', 'key2', 'key'], list(d))
+        assert ['key', 'key2', 'key'] == list(d)
 
     def test__iter__types(self):
         d = self.make_dict([('key', 'one'), ('key2', 'two'), ('key', 3)])
         for i in d:
-            self.assertTrue(type(i) is self.key_cls, (type(i), self.key_cls))
+            assert type(i) is self.key_cls, (type(i), self.key_cls)
 
     def test_keys__contains(self):
         d = self.make_dict([('key', 'one'), ('key2', 'two'), ('key', 3)])
-        self.assertEqual(list(d.keys()), ['key', 'key2', 'key'])
+        assert list(d.keys()) == ['key', 'key2', 'key']
 
-        self.assertIn('key', d.keys())
-        self.assertIn('key2', d.keys())
+        assert 'key' in d.keys()
+        assert 'key2' in d.keys()
 
-        self.assertNotIn('foo', d.keys())
+        assert 'foo' not in d.keys()
 
     def test_values__contains(self):
         d = self.make_dict([('key', 'one'), ('key', 'two'), ('key', 3)])
-        self.assertEqual(list(d.values()), ['one', 'two', 3])
+        assert list(d.values()) == ['one', 'two', 3]
 
-        self.assertIn('one', d.values())
-        self.assertIn('two', d.values())
-        self.assertIn(3, d.values())
+        assert 'one' in d.values()
+        assert 'two' in d.values()
+        assert 3 in d.values()
 
-        self.assertNotIn('foo', d.values())
+        assert 'foo' not in d.values()
 
     def test_items__contains(self):
         d = self.make_dict([('key', 'one'), ('key', 'two'), ('key', 3)])
-        self.assertEqual(list(d.items()),
-                         [('key', 'one'), ('key', 'two'), ('key', 3)])
-        self.assertEqual(list(d.items()),
-                         [('key', 'one'), ('key', 'two'), ('key', 3)])
+        expected = [('key', 'one'), ('key', 'two'), ('key', 3)]
+        assert list(d.items()) == expected
+        assert list(d.items()) == expected # was the repetition intentional?
```

Comment:
@asvetlov I stumbled upon this repeated assertion. Either it's redundant or I don't understand it's meaning. It seems to have originated at f84e6bc6 . Should it stay or should it go?